docs/tutorials/podman-install-certificate-authority.md

```diff
@@ -0,0 +1,75 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+Install Certificate Authority
+=============================
+
+Organizations may create their own local certificate authority (CA) or acquire one from a third party.  This may mean more than one certificate such as one or more intermediate certificates and a root certificat for example.  In any case, there is a need to add the certificate authority (CA) certificate(s) such that it can be employed for various use cases.
+
+### Method one
+
+The following steps are one method to add such certificates to podman.  It is assumed that podman is running and the certificate(s) to be installed are available on an accessible server via curl.  If such access is not possible, an alternative method follows.  Certificates may be either individual or concatenated (bundles).
+
+First, assuming a running podman machine, ssh into the machine:
+```
+podman machine ssh
+```
+
+Next while in the machine, change to the directory where the certificate(s) should be installed:
+```
+[root@localhost ~]# cd /etc/pki/ca-trust/source/anchors
+```
+
+Then use curl to download the certificate.  Notes:
+* the -k is only necessary if connecting securely to a server for which the certificate is not yet trusted
+* the MY-SERVER.COM/SOME-CERTIFICATE.pem should be replaced as appropriate
```

Comment:
suggestion for consideration.  I'd captialize the 't" in the first word "the" in both bullet points.

Reply:
done